strawberry/codegen/query_codegen.py

```diff
@@ -504,7 +504,9 @@ def _field_from_selection_set(
         selected_field = self.schema.get_field_for_type(
             selection.name.value, parent_type.name
         )
-        assert selected_field
+        assert (
+            selected_field
+        ), f"Couldn't find {parent_type.name}.{selection.name.value}"
```

Comment:
I hit this assertion a bunch when testing and I make this change every single time and revert before pushing it up.  This error actually gets exposed to the user when they misspell a field name in the GraphQL query, so it makes sense to me to at least give it a more helpful error message so that the user knows something about what the codegen was looking for.

It would probably be better to _not_ use `assert` for this at some point and raise a nicer exception overall, but that felt more invasive than I wanted for this particular PR.

Reply:
let's keep the assert, and improve the message in another PR 😊